crates/gpui/src/platform/windows/direct_write.rs

```diff
@@ -792,9 +785,9 @@ impl DirectWriteState {
     }
 
     fn raster_bounds(&self, params: &RenderGlyphParams) -> Result<Bounds<DevicePixels>> {
-        let glyph_analysis = self.create_glyph_run_analysis(params)?;
+        let glyph_analysis = self.create_glyph_run_analysis(params, !params.is_emoji)?;
 
-        let bounds = unsafe { glyph_analysis.GetAlphaTextureBounds(DWRITE_TEXTURE_CLEARTYPE_3x1)? };
+        let bounds = unsafe { glyph_analysis.GetAlphaTextureBounds(if params.is_emoji { DWRITE_TEXTURE_CLEARTYPE_3x1 } else { DWRITE_TEXTURE_ALIASED_1x1 })? };
```

Comment:
Direct2D always renders emoji with grayscale antialiasing. We shouldn't use ClearType for this

Reply:
why would alpha correction / contrast adjustment be needed in case where we are compositing a color glyph and not just fonts? edges shouldn't be a big concern there or am i looking at this wrong

Reply:
A COLRv0 glyph consists of regular monochrome font glyphs layered on top of each other. The font provides a solid color fill that should be used for each layer. So, drawing the glyph for each layer is no different from drawing regular text with some foreground color

Reply:
Like, whether we want to use ClearType or grayscale to generate the monochrome raster for each layer is a separate matter. (And we don't want to use ClearType)